src/components/pages/Home/_MainHeader.less

```diff
@@ -6,4 +6,18 @@
     background: white;
     box-shadow: 0px 2px 5px 0px #00000033;
     padding: 10px 20px 5px;
-}  
\ No newline at end of file
+    width: 100vw;
+    display: flex;
+    justify-content: space-between; 
+}  
+
+.user-button{
+    border: 1px rgb(97, 92, 92) solid;
+    box-shadow: 0px 2px 5px 0px #00000033;
+    height: 27px; 
+    width: 27px;
+    padding: 3px 5px; 
+    margin-right: 50px;
+    margin-top: 20px;
+    border-radius: 50%;
+}
```

Comment:
Having this in less is going to be a big help when styling. Thanks for embracing the switch previously so we have this ready to go!